Detectors/CPV/reconstruction/include/CPVReconstruction/RawDecoder.h

```diff
@@ -101,17 +100,17 @@ class RawDecoder
   ///
   /// Performing various consistency checks on the RCU trailer
   /// In case of failure an exception is thrown.
-  void checkRCUTrailer();
+  //void checkRCUTrailer();
 
-  void addDigit(uint32_t padWord, short ddl);
+  void addDigit(uint32_t padWord, short ddl, uint16_t bc);
```

Comment:
Do I understand correctly that each digit now carries BC index? This is useless information: you should define trigger records which keep info about BC and point to the first and last digit belonging to the event. Thus you reduce data size by ~50%

Reply:
@peressounko, this is a bit stupid, I understand. However to create TriggerRecord one need to know orbit and bc. Previously this info was available in RDH. Now RDH contains only orbit info, bc is coded in the payload. Otherwise I need to heavily change structure of program and produce TriggerRecords right in RawDecoder. I'll consider this possibility.

Reply:
@sevdokim, anyway for TriggerRecord you need to know amount of digits => when you will fill it you will know orbit and BC.

Reply:
Dear @peressounko, addDigit() produces not o2::cpv::Digits but some temporary digit-like objects. Then RawToDigitConverterSpec.cxx produces Digits from those objects and TriggerRecords from rdh. Now schema is changed. I think that best way will be to produce also temporary TriggerRecord-like objects (with BCs) and provide those together to RawToDigitConverterSpec.cxx.